lona/view.py

```diff
@@ -45,24 +38,6 @@ def __init__(
         self._view_runtime: ViewRuntime = view_runtime
         self._request: Request = request
 
-    # objects #################################################################
-    @classmethod
-    def iter_objects(cls: type[V]) -> Iterator[V]:
-        # TODO: remove after 1.8
-
-        warnings.warn(
-            'LonaView.iter_objects() will be removed in 1.8',
```

Comment:
Don't we need to remove this from docs as well?

Reply:
I will leave the docs for this method in the docs until 2.0, but i will add ``note: removed in 1.8`` like before. I do this usually when writing the changelog.